src/concrete/ml/quantization/quantized_module.py

```diff
@@ -142,6 +146,14 @@ def __init__(
         # Input-output quantizer mapping for composition is not enabled at initialization
         self._composition_mapping: Optional[Dict] = None
 
+        # Initialize _preprocessing_numpy
+        # The onnx graph is used to pre-process the inputs before FHE execution
+        self._preprocessing_module = NumpyModule(onnx_preprocessing) if onnx_preprocessing else None
+
+        # Ensure that there is no preprocessing step
+        if self._preprocessing_module is not None:
```

Comment:
is this always true ? why ?

Reply:
I have lost track here. We create a `QuantizedModule` object and we CAN pass or not an `onnx_preprocessing`. Its typing is

`onnx_preprocessing: Optional[onnx.ModelProto] = None,`

so yes it can be None. The line above does

```python
        self._preprocessing_module = NumpyModule(onnx_preprocessing) if onnx_preprocessing else 
```

Reply:
ok sorry I think last time I must have read `if self._preprocessing_module.onnx_preprocessing is None` instead of `if self._preprocessing_module is not None:`, which made less much sense 😅  Thanks for explaining it ! looks good